â–¿ Result<String, DataError>
  - success: "Success"
